### PR TITLE
PLT-339: use Dupable in inliners

### PR DESCRIPTION
**Pull request by kwxm**
PLT-339.  This seemed suspiciously easy.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting master unless this is a cherry-pick backport
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs [hunk 1]
```diff
@@ -273,7 +274,7 @@ processTerm = handleTerm <=< traverseOf termSubtypes applyTypeSubstitution where
         _    -> typeSubstTyNamesM substTyName t
     -- See Note [Renaming strategy]
     substTyName :: tyname -> InlineM tyname name uni fun a (Maybe (Type tyname uni a))
-    substTyName tyname = gets (lookupType tyname) >>= traverse PLC.rename
+    substTyName tyname = gets (lookupType tyname) >>= traverse liftDupable
```
Comment (kwxm) [inline plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs:277]: `liftDupable` is perhaps not as meaningful a name as `rename`.

**Review by michaelpj — APPROVED** [review]
I was expecting it to be easy.

**Review by effectfully — COMMENTED** [review]
I believe it was Confucius who once said that post-merge approval is the dual of self-fulfilling prophecy.

**Comment by kwxm** [comment]
> I believe it was Confucius who once said that post-merge approval is the dual of self-fulfilling prophecy.

I don't think it really needed three reviewers: that was mostly just FYI for people who might care.

**Pull request merged**